### DISCUSSION TITLE
app router glitch on all client component page

### DISCUSSION THREAD

**Discussion by fiqryq** [vercel/next.js#56985] [App Router]
I have some issues when using the app router. Every page in my client components is always being rerendered when I perform actions like input, clicking buttons, and making state changes. This issue occurs on all my pages. Can someone provide a solution for my problem?

glitch example :

https://github.com/vercel/next.js/assets/25787603/66a6d011-89d1-4718-b472-b4652e4725a2

**Comment by icyJoseph** [comment]
Got some code to show? Repository link?

**Reply by fiqryq** [reply]
its my sample project https://github.com/fiqryq/gpt-clone 

this input also glitch.
![image](https://github.com/vercel/next.js/assets/25787603/9d4e0268-3489-45f5-acfc-32f8269059fc)
